blocks/post-sidebar/post-sidebar.js

```diff
@@ -56,7 +81,7 @@ export default async function decorate(block) {
 
   const picMedia = [{ media: '(min-width: 160px)', width: '160' }];
   const pic = createOptimizedPicture(authorImage, '', false, picMedia);
-  block.innerHTML = `<a class="author-image" href="${authorUrl}">${pic.outerHTML}</a>
+  block.innerHTML = `<a class="author-image" href="${encodeURIComponent(authorUrl)}">${pic.outerHTML}</a>
```

Comment:
don't think you need `encodeURIComponent` here. I don't think it will cause any problems, but since it's a full path and not just a param, probably not required.

Reply:
fair.  Removing along with some minor css changes.